### ISSUE TITLE
Postgres.app will not launch when installed to `/Users/xxxxxxx/Applications`

### ISSUE THREAD

**Issue by rbpatt2019** [PostgresApp/PostgresApp#562]
**TLDR: Can Postgres.app be installed in a user specified location, or does it require an install to the root `/Applications`directory?**

After a clean install following the instructions [here](https://postgresapp.com/documentation/install.html), I double click on the Postgres.app icon in the Applications folder and...nothing happens. 

That's not entirely true. If I pay close attention, I can see a small blip in the tray, which makes it appear as if the app is opening and instantly closing. 

Some additional information:
1. I can't drag the Postgres.app icon to the Applications folder icon in the .dmg window as that requires Admin privileges, which I don't have and can't get (I'm on a work computer). So, I'm having to drag it to my local Applications folder, `/Users/xxxxxxx/Applications`.

1. After adding `psql` to my path (by altering it in my profile files as , again, I don't have sudo access on this computer), I get: 
```bash
$ which psql
/Users/xxxxxxx/Applications/Postgres.app/Contents/Versions/latest/bin/psql
```
as expected, since it's installed there. However, when I attempt to issue `psql`, I see the following
```bash
$ psql --version
dyld: Library not loaded: /Applications/Postgres.app/Contents/Versions/12/lib/libpq.5.dylib
  Referenced from: /Users/xxxxxxx/Applications/Postgres.app/Contents/Versions/latest/bin/psql
  Reason: image not found
[1]    46959 abort      psql
```

I've noticed that it is complaining that it can't load the library in the `/Applications` folder, rather than the `/Users/xxxxxxx/Applications` directory where I installed it. This leads me to believe that Postgres.app requires a global install, rather than a user-specific one. 

Which brings me to my questions:
1) Is it possible to do a user-specific install, rather than installing into the root `/Applications` directory?
2) Can Postgres.app be installed without admin privileges (required to drag the app icon to the dir icon in dmg window) or sudo privileges (recommended method of setting the path)?

Thanks for the help!

**Comment by jakob** [comment]
Short answer: Postgres.app needs to be in /Applications

Slightly less short answer: All libraries and binaries in Postgres.app are linked with absolute paths. This is necessary because otherwise you can't build things that link against libpq.

If you really wanted to, you could use install_name_tool to change all the library paths to a custom location, but that would be a lot of effort and break the code signature, so you might have to re-sign the app as well.
